code/modules/projectiles/ammunition/ballistic/shotgun.dm

```diff
@@ -134,10 +134,10 @@
 
 /obj/item/ammo_casing/shotgun/dart
```

Comment:
The bioterror dart (below this) is initializing with 30 units total, either those darts should keep the 30 unit amount (in which case, the description has to be changed too), or their chemicals should be lowered.

Reply:
i feel like them keeping their 30u chem total would be fine, since its a nukie item right?